docs/sql/cli.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: default
+title: SQL CLI
+parent: SQL
+nav_order: 20
+---
+
+# SQL CLI
+
+SQL CLI is a stand-alone Python application that you can launch with the `odfesql` command.
+
+Install the SQL plugin to your Elasticsearch instance, run the CLI using MacOS or Linux, and connect to any valid Elasticsearch end-point.
+
+![SQL CLI](../../images/cli.gif)
+
+## Features
+
+SQL CLI has the following features:
+
+- Multi-line input
+- Autocomplete for SQL syntax and index names
+- Syntax highlighting
+- Formatted output:
+  - Tabular format
+  - Field names with color
+  - Enabled horizontal display (by default) and vertical display when output is too wide for your terminal, for better visualization
+  - Pagination for large output
+- Works with or without security enabled
+- Supports loading configuration files
+- Supports all SQL plugin queries
+
+## Install
+
+To install the CLI:
+
+1. We suggest you install and activate a python3 virtual environment to avoid changing your local environment:
+
+```
+pip install virtualenv
+virtualenv venv
+source ./bin/activate
+```
+
+2. To install the CLI:
+
+```
+pip install odfesql
+```
+
+The SQL CLI only works with Python 3.
+{: .note }
+
+## Using the CLI
+
```

Comment:
Should we add another step "launch local ES instance and install SQL plugin" as the 1st step? Or we could let user know that this example is based on the fact that running `odfesql` will only connect to `http://localhost:9200`

Reply:
Added.